ipfs-cluster-ctl/main.go

```diff
@@ -166,6 +166,42 @@ is also TODO
 					Name:  "recursive, r",
 					Usage: "add directory paths recursively, default false",
 				},
+				cli.BoolFlag{
+					Name:  "shard",
+					Usage: "break the file into pieces (shards) and distributed among peers, default false",
+				},
+				cli.BoolFlag{
+					Name:  "quiet, q",
+					Usage: "write minimal output",
+				},
+				cli.BoolFlag{
+					Name:  "silent",
+					Usage: "write no output",
+				},
+				cli.BoolFlag{
```

Comment:
string flag. Mention options in help

Reply:
Making trickle a bool means that `ipfs-cluster-ctl add` has subtlely different flag semantics than go-ipfs.  I agree a string flag for layout makes more sense in the long run but it comes with the potential downside of making our UI a little more annoying to use for people switching between our CLI and go-ipfs.  I'm fine with doing this just wanted to make the consequences of our choice here explicit.